### PR TITLE
Fix, change network.disable.rpfilter type from integer to boolean.

### PR DESCRIPTION
**Pull request by soreana**
### Description

Currently it is not possible to set the `network.disable.rpfilter` global setting to `false` or `true`. It only accepts integer value which is incorrect.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### How Has This Been Tested?

Login to the UI, change try to change the `network.disable.rpfilter` global setting value. It only accepts Integer which is not correct. It should accept `true` or `false`.

This issue will be fixed after this pr.

### CODE REVIEW

File: server/src/main/java/com/cloud/configuration/Config.java [hunk 1]
```diff
@@ -265,7 +265,7 @@ public enum Config {
     NetworkRouterRpFilter(
             "Network",
             ManagementServer.class,
-            Integer.class,
+            Boolean.class,
```
Comment (sureshanaparti) [inline server/src/main/java/com/cloud/configuration/Config.java:268]: Upgrading any existing env with current int value should change value to true / false.

Comment (soreana) [reply server/src/main/java/com/cloud/configuration/Config.java:268]: Env with an int value should change to false.

**Comment by nvazquez** [comment]
Nice catch @soreana - reviewing your PR have found out the same happens for 'consoleproxy.disable.rpfilter', can you please add this one as well?

**Comment by nvazquez** [comment]
@blueorangutan package

File: server/src/main/java/com/cloud/configuration/Config.java (hunk 1 above)
Comment (sureshanaparti) [reply server/src/main/java/com/cloud/configuration/Config.java:268]: @nvazquez Agree that both are enabled when the value is 'true', and also, the default value set is 'true'. As the config is set to integer type, the update config fails to take the boolean values, and only accepts integer values. If anyone have update the config with integer value, then it is disabled, so upgrade path should set to 'false' accordingly. What is your opinion?

```
(test-env) 🐱 > list configurations name=rpfilter filter=name,value
{
  "configuration": [
    {
      "name": "consoleproxy.disable.rpfilter",
      "value": "true"
    },
    {
      "name": "network.disable.rpfilter",
      "value": "true"
    }
  ],
  "count": 2
}
(test-env) 🐱 > update configuration name=network.disable.rpfilter value=false
🙈 Error: (HTTP 431, error code 4350) There was error in trying to parse value: false. Please enter a valid integer value for parameter network.disable.rpfilter
(test-env) 🐱 > 
(test-env) 🐱 > update configuration name=network.disable.rpfilter value=3
{
  "configuration": {
    "category": "Network",
    "description": "disable rp_filter on Domain Router VM public interfaces.",
    "isdynamic": false,
    "name": "network.disable.rpfilter",
    "value": "3"
  }
}
(test-env) 🐱 > 
(test-env) 🐱 > update configuration name=consoleproxy.disable.rpfilter value=true
🙈 Error: (HTTP 431, error code 4350) There was error in trying to parse value: true. Please enter a valid integer value for parameter consoleproxy.disable.rpfilter
(test-env) 🐱 > 
(test-env) 🐱 > update configuration name=consoleproxy.disable.rpfilter value=false
🙈 Error: (HTTP 431, error code 4350) There was error in trying to parse value: false. Please enter a valid integer value for parameter consoleproxy.disable.rpfilter
(test-env) 🐱 > 
(test-env) 🐱 > update configuration name=consoleproxy.disable.rpfilter value=9
{
  "configuration": {
    "category": "Console Proxy",
    "description": "disable rp_filter on console proxy VM public interface",
    "isdynamic": false,
    "name": "consoleproxy.disable.rpfilter",
    "value": "9"
  }
}
(test-env) 🐱 > 
(test-env) 🐱 > 
(test-env) 🐱 > list configurations name=rpfilter filter=name,value
{
  "configuration": [
    {
      "name": "consoleproxy.disable.rpfilter",
      "value": "9"
    },
    {
      "name": "network.disable.rpfilter",
      "value": "3"
    }
  ],
  "count": 2
}
(test-env) 🐱 > 
```

**Comment by nvazquez** [comment]
@blueorangutan test

File: server/src/main/java/com/cloud/configuration/Config.java (hunk 1 above)
Comment (soreana) [reply server/src/main/java/com/cloud/configuration/Config.java:268]: @sureshanaparti I'm totally agree with you. I don't know how I can achieve that goal. Do you have any suggestion?

Comment (sureshanaparti) [reply server/src/main/java/com/cloud/configuration/Config.java:268]: @soreana Try updating the config value to 'false' in the upgrade path (either Upgrade class / schema file) when the value is not 'true'.

**Review by rohityadavcloud — APPROVED** [review]

**Review by nvazquez — APPROVED** [review]

**Comment by nvazquez** [comment]
@soreana @sureshanaparti the upgrade path for integer values (if any) can be fixed in the next milestone

**Comment by soreana** [comment]
I've created https://github.com/apache/cloudstack/issues/6390 issue as a reminder.

**Pull request merged**